### DISCUSSION TITLE
[Script request] Tube Archivist

### DISCUSSION THREAD

**Discussion by Mightyretro** [community-scripts/ProxmoxVE#459] [Request script] [answered]
### Application Name

Tube Archivist

### Website

https://github.com/tubearchivist/tubearchivist

### Description

Hi,

love to see a script for this project come alive

Kind regards

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by frezeen** [comment]
yes, i agee to request

edit: a guide for to setup a proxmox LXC for tubearchivist

**Comment by DeclanE47** [comment]
This would be a fantastic addition

**Comment by MickLesk** [comment, marked as answer]
**Update - 20-02-2025:** 
Deferred, zipfile doesnt contain frontend/backend, no automatic build possible

**Reply by Data-PhiIe** [reply]
Can you use the existing docker image?
https://hub.docker.com/r/bbilly1/tubearchivist
the YAML file:
https://github.com/tubearchivist/tubearchivist/blob/master/docker-compose.yml

**Reply by MickLesk** [reply]
Nope its too Tricky to maintain

**Reply by Data-PhiIe** [reply]
Hello, I wanted to share some more information! I don't think the project no longer needs to be deferred. I checked with the tubearchivist devs on discord; apparently this was deferred during their project freeze the files weren't available during that time, They said they are now. I really want tubearchivist. And since I'm not able to contribute to writing the script; at thew very least I figured I could try to communicate between the 2 development groups and Iron this out, hope I was able to help! :)


https://github.com/tubearchivist/tubearchivist/releases

**Reply by MickLesk** [reply]
But there are no release files...?

**Reply by Data-PhiIe** [reply]
Wait still not there? They said they were available now when I asked on their discord, Let me check with them.

**Reply by Data-PhiIe** [reply]
quoting the mod that responded on discord

"This looks like it was deferred during our project freeze. Project freeze was lifted with the release of v0.5.0. Those files are included in our releases. https://github.com/tubearchivist/tubearchivist/releases

I have confirmed that in earlier and current versions, backend and frontend files existed, they just weren't explicitly called out (this is changed in v0.5.0)."

**Reply by lamusmaser** [reply]
What release files are you looking for? Source code zip of each release is produced with each release version.

**Reply by tremor021** [reply]
![image](https://github.com/user-attachments/assets/0731c268-f03a-4128-ad9d-b21c064bd470)
Also this... We wont be installing it like this. Once they have a stable release that can be simply installed and not jump through hoops for the initial install, we might do it. its a simple app

**Reply by MickLesk** [reply]
For 0.5.0 it doesnt exist an Release File too ^^

**Comment by frezeen** [comment]
sorry  if i take this req up, 0.5.8 zip has fronted e backed files... or am i wrong?

**Reply by MickLesk** [reply]
yeah, looks so. @community-scripts/contributor anyone? @CrazyWolf13 you are the right one for yt specials :D

**Reply by CrazyWolf13** [reply]
@MickLesk Not me on this one, that was of the most unfriendliest dev I ever encountered, als the app takes a enormous amount of ressources.

**Reply by MickLesk** [reply]
ah right, i remember, okay if i have time in december, i can take a look and give feedback here

**Reply by Data-PhiIe** [reply]
looks like front/back end are available now? I could be mistaken maybe double check my statement here https://github.com/tubearchivist/tubearchivist

**Comment by Bensonheimer992** [comment]
Are there any updates to this ?

**Reply by MickLesk** [reply]
Nope

**Reply by Data-PhiIe** [reply]
I was very surprised to wake up one morning and see a lot of script requests, including this one, that have been in limbo have been added! In my head I had chalked these things getting put on the back burner/deprioritized up to a lack of resources. I had told myself "it's gonna be a while" and had mentally given up on seeing them any time soon. I'm super happy to have that negative thought shattered by the sudden rapid influx of scripts that were requested and increase in pacing for seeing them added to the site. Thank you to everyone contributing to helper scripts, y'all are doing amazing work! :)
